libs/ui-watt/src/lib/components/table/watt-table.component.ts

```diff
@@ -328,6 +340,14 @@ export class WattTableComponent<T>
   _getColumnCell(column: KeyValue<string, WattTableColumn<T>>, row: T) {
     return column.value.cell?.(row) ?? this.getCellData(row, column.value);
   }
+
+  /** @ignore */
+  _isActiveRow(row: T) {
+    if (!this.activeRow) return false;
+    return this.activeRowComparator
+      ? this.activeRowComparator(row, this.activeRow)
+      : row === this.activeRow;
+  }
```

Comment:
passing by comment :D 

suggestion: I don't know the reason for this change, but is it not possible to use [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) to check for active row? Set stores unique values so saving the active row in the Set, and clearing it before saving the next row sounds like a way to make sure that a Set contains only one unique row. Bye :D

Reply:
I'm not sure I fully understand your suggestion, but I don't think `Set` solves the problem here. The reason I added the custom row comparator input is because, in wholesale, we use the router to store the currently active "row" (actually just an ID), while the table data is saved in the store for the entire lifecycle of the page. So when we navigate back to the table from a subpage, we fetch a new "row" object based on the ID in the router. That row object is never the same as any of the rows in the table, even though the properties are exactly the same. So to fix that, we still update the active row input with new row, but also tell the table to compare ID's instead of the entire object. Does that make sense?

Reply:
I'm resolving this just so I can merge it now, but would love to continue the discussion afterwards 🙂

Reply:
Sure. As I said, a quick passing by comment. Didn't have enough context to understand the solution properly. Thanks for sharing some insights. We can discuss it some day :)